### DISCUSSION TITLE
Portable pdb sourcefile notation

### DISCUSSION THREAD

**Discussion by bpostaci** [icsharpcode/ILSpy#2611] [Help]
When i generated a portable pdb then debug it in debugger noticed that if we have source files in nested folders architecture like boards\pcb\v3\mcu.cs , 
In callstack for related frame it shown as boards.pcb.v3\mcu.cs
Debugger looking for a path actually does not exist to open source code.
Is there any setting that we can generate pdb in nested folder style ?

**Comment by siegfriedpammer** [comment]
ILSpy generates portable PDBs with the source code embedded in the PDB file, so there should not be any need for Visual Studio to go look for a file path, as it should just extract the embedded sources stream to a temporary folder and use it from there.

Would it be possible for you to help us investigate the issue by providing a minimal repro? Thanks!

**Reply by bpostaci** [reply]
I think i found what setting i missed UseNestedDirectoriesForNamespaces
